application/app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,13 @@
     <string name="forgot_password">Forgot password?</string>
     <string name="or">Login</string>
     <string name="continue_without_account">Continue without account</string>
+    <!-- Strings related to login -->
+    <string name="prompt_email">Email</string>
+    <string name="prompt_password">Password</string>
+    <string name="action_sign_in">Sign in or register</string>
+    <string name="action_sign_in_short">Sign in</string>
+    <string name="welcome">"Welcome!"</string>
+    <string name="invalid_username">Not a valid username</string>
+    <string name="invalid_password">Password must be >5 characters</string>
```

Comment:
Can we replace `Password must be >5 characters` to `Password length must be greater than 5 letters?`. This is more user friendly. What do you think?